src/components/ParserComponent.tsx

```diff
@@ -164,22 +227,34 @@ export default class ParserComponent extends React.Component<ParserInterface,
    * Purpose: attempts to parse the file uploaded by the user.
    * @param {Object} fileEvent: the event passed into this component
    */
-  parse(fileEvent: any) {
-    // this.isValid(fileEvent);
-    // this.sortData(this.state.data);
-    // this.inferTypes(this.state.data);
+  async parse(fileEvent: any) {
+    this.setState(() => {
+      return {
+        showTimeline: false,
```

Comment:
show timeline instantiates to false and then is true if the file is a csv? What if it is not a csv?